upd_script/wifi/cinch_uninstaller.sh

```diff
@@ -0,0 +1,89 @@
+if [[ $EUID -ne 0 ]]; then
+	echo "This script must be run as root"
+	exit 1
+fi
+
+SCRIPTDIR="$(readlink -f $(dirname $0))"
+
+# check whether cinch is already installed
+if [[ ! -f /home/pi/cinch ]]; then
+  exit 1
+fi
+
+# stopping services
+# wifi_log.service is for cinch -> service to run the wifi debug script and creates an appending log in /boot/logs folder
+# channel_select.service is for cinch
+# hostapd is for cinch -> for configuring the hotspot details (such ssid, channel, PSKs, etc)
+# dnsmasq is for cinch -> provides dns, dhcp and network boot functionalities
+systemctl stop wifi_log.service
+systemctl stop channel_select.service
+systemctl stop dnsmasq.service
+systemctl stop hostapd.service
+
+# then we need to disable them
+systemctl disable wifi_log.service
+systemctl disable channel_select.service
+# warnings come from here -> it's related to the runlevel from which we stop the service(s)
+systemctl disable dnsmasq.service
+systemctl disable hostapd.service
+
+# remove configuration files needed for cinch to work
+rm -f /etc/systemd/system/wifi_log.service
+rm -f /etc/systemd/system/channel_select.service
+rm -f /etc/dnsmasq.d/dnsmasq.conf
+rm -f /etc/dnsmasq.d/cinch.conf
+rm -f /etc/sysctl.d/cinch.conf
+rm -f /etc/modprobe.d/cinch-blacklist.conf
+
+# check if a backup was made
+if [[ -f /etc/hostapd/hostapd.conf.cinch.bak ]]; then
+  # and if so, then rename the backup to the original filename
+  mv -f /etc/hostapd/hostapd.conf.cinch.bak /etc/hostapd/hostapd.conf
```

Comment:
Okay its fine to have it then, its upto you. Second possibility is quite rare since they will need our RTL wifi adapter to have this working. So its fine to have it or not.